clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -32,18 +32,18 @@ import {TelemetryService} from "./services/reporter/TelemetryService";
 import {createFileWithGivenPath, initializeSettings} from "./services/Settings";
 import {ZoweApi} from "./services/ZoweApi";
 
+const zoweApi: ZoweApi = new ZoweApi();
+const profileService: ProfileService = new ProfileService(zoweApi);
+const copybooksPathGenerator: CopybooksPathGenerator = new CopybooksPathGenerator(profileService);
+const copyBooksDownloader: CopybookDownloadService =
+new CopybookDownloadService(zoweApi, profileService, copybooksPathGenerator);
```

Comment:
Could you please keep in on one line? Or add indentation on 39 line.

Reply:
done